apps/coscrad-frontend/src/app/bootstrap-index-to-detail-flow-routes.tsx

```diff
@@ -17,16 +18,17 @@ export const bootstrapIndexToDetailFlowRoutes = ({
 }: ConfigurableContent): CoscradRoute[] => {
     const resourceTypesAndRoutes = indexToDetailFlows
```

Comment:
Add a TODO to override with the dynamic info from the resource info endpoint via Redux.

Reply:
[Here](TODO[https://www.pivotaltracker.com/story/show/185338095) is a story for this.